### PR TITLE
UTOPIA-1006: PIA subHeader - submit button UI issue when disabled

### PR DESCRIPTION
**Pull request by Bottle7**
# Description

This PR includes the following proposed change(s):

- Fix 'Submit' button disabled pointer

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Tested under repro steps.

## Development Dependency Working Agreement
- [ ] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [x] New and existing unit tests pass locally with my changes
- [x] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

**Review by binggaofw — APPROVED** [review]

File: src/frontend/src/sass/_buttons.scss [hunk 1]
```diff
@@ -101,7 +101,7 @@ a:hover {
   --text-color: white;
 
   border: 2px solid $bg-disabled;
-  cursor: not-allowed;
+  cursor: not-allowed !important;
   text-decoration: none;
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_buttons.scss:105]: I see text decoration is taking other properties as well. Try adding "important" in dev and observe the difference.

```suggestion
  text-decoration: none !important;
```

File: src/frontend/src/sass/_buttons.scss [hunk 2]
```diff
@@ -30,7 +30,7 @@ a:hover {
   -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
   cursor: pointer;
   transition: border-color 0.25s;
-
+
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_buttons.scss:33]: unnecessary space

**Review by binggaofw — APPROVED** [review]

**Review by antsand — APPROVED** [review]
Looks Good!!

**Review by binggaofw — APPROVED** [review]

**Review by kushal-arora-fw — APPROVED** [review]
Cool

**Pull request merged**